docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -37,6 +37,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 | Draft 1.7 | 27 March 2025     | Update FC review policy           |
 | Draft 1.8 | 01 April 2025     | Update I2C Device Info            |
 | Draft 1.9 | 12 September 2025 | Update motor requirements         |
+| Draft 2.0 | 12 November 2025  | GPIO usage clarifications         |
```

Comment:
From a decimal standpoint this is completely logical of c., but seeing as most of the changes are (I think) relatively minor or more or less clarifications of already existing requirements, does `1.10` from a sematic standpoint not make more sense?


```suggestion
| Draft 1.10 | 12 November 2025  | GPIO usage clarifications         |
```

Reply:
I don't think it really makes a difference. Version 1.0 was not significant.